bin/acceptance-test

```diff
@@ -1,24 +1,77 @@
 #!/usr/bin/env bash
 
-test_equality() {
-    if [ "$1" = "$2" ]; then
-      echo "The server gave an expected response"
-      exit 0
-    else
-      echo "The server gave an unexpected response"
+set -e
+
+function start {
+  java -jar $1 >/dev/null &
+  echo $!
+}
+
+function wait_for_server {
+  local host=$1
+  local port=$2
+
+  echo "Waiting for server to start"
+  local max_attempts=5
+  echo "$max_attempts"
```

Comment:
**question** There are quite a few new `echo` statements. Are all of these valuable?

```bash
echo $max_attempts
# ...

echo "Entered the loop"

# ...

echo "connected"

# ...
```

Reply:
Forgot to remove some of the echo statements, thanks for catching that!